flink-state-backends/flink-statebackend-heap-spillable/pom.xml

```diff
@@ -37,5 +37,13 @@ under the License.
 	<packaging>jar</packaging>
 
 	<dependencies>
+		<!-- core dependencies -->
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
```

Comment:
now that we have a scala dependency we have to add the scala suffix to the module again. (The suffix was removed since there were no scala dependencies before, in which case we forbid scala suffixes)